core/services/PolicyService.js

```diff
@@ -1,137 +1,137 @@
 module.exports = {
-  /**
```

Comment:
Please remove unneeded spaces at the beginning of the strings. We still use only 2 spaces per each nesting level.